core/src/main/java/com/yahoo/oak/OakMapBuilder.java

```diff
@@ -29,6 +29,7 @@
     private int chunkMaxItems;
     private long memoryCapacity;
     private BlockMemoryAllocator memoryAllocator;
+    private Long preferredBlockSize;
```

Comment:
long->int, specifically when able to be set from outside.

Reply:
:+1: